src/rust/ensogl/src/display/scene.rs

```diff
@@ -539,9 +539,11 @@ pub struct Callbacks {
 // === Renderer ===
 // ================
 
+pub type RendererLogger = disabled::Logger;
+
 #[derive(Clone,CloneRef,Debug)]
 pub struct Renderer {
-    logger    : Logger,
+    logger    : RendererLogger,
```

Comment:
Use `Logger` when storing fields.
`RendererLogger` on the other hand may be defined somewhere upper in the crate, so we have a place where we can easily toggle different logger types behavior.